source/deployment/enterprise-deployment-guide.rst

```diff
@@ -0,0 +1,216 @@
+
+Taking this a step further, the emergence of ChatOps means Mattermost is no longer an exclusive developer tool. In fact, it has the ability to enhance communication through a multitude of integrations, extensions, and customization.
+
+Here’s just a sample of available integrations our customers are deploying:
+
+- Confluence
+- Docker
+- GitHub / GitLab
+- Jira
+- Outlook
+- Trello
+
+Taking this further, because Mattermost is an Open Source application, it can be completely customized, in every way, to suit your organizational needs.
+
+***************
+Further Reading
+***************
+
+Mattermost Licensing
+--------------------
+
+Mattermost Enterprise Edition comes with two licensing options: Enterprise E10 and Enterprise E20 which can be tested with a `Free 30-Day Trial <https://mattermost.com/trial/>`_. We tailored the E10 licence to suit smaller organizations with less need for automation and compliance, where our E20 license will provide you with features like ADFS SAML 2.0 support, automated compliance export and team specific permissions. You can find a full comparison `here <https://mattermost.com/pricing-feature-comparison>`_.
+
+
+.. Note::
+    Reach out to our Sales team by contacting sales@mattermost.com to upgrade your license at any time.
+
+Mattermost Server Environment
+-----------------------------
+
+At its core, Mattermost is an open source, hybrid-cloud alternative to proprietary SaaS messaging for teams. It is designed to increase the agility, efficiency and innovation in high trust organizations while keeping data and operations under IT control. Mattermost ensures you can own your data. An important step is to determine the environment for the Mattermost server.
+
+We officially support running the Mattermost Server on multiple `Linux distributions, Docker and on-premises cloud solutions <https://docs.mattermost.com/guides/administrator.html#installing-mattermost>`_.
+
+.. Note::
+    Other Linux distributions might work too, but are not officially supported.
+
+Client Usage
+------------
+
+Depending on your environment and your users, you have several options when deploying the Mattermost Client: Web Browser, `Desktop <https://docs.mattermost.com/install/desktop.html>`_, `Mobile <https://docs.mattermost.com/mobile/mobile-overview.html>`_ or all three. We `provide binaries <https://mattermost.com/download/#mattermostApps>`_ for MS Windows, macOS, Linux, iOS and Android. However, depending on your organization policies deployment on these platforms can vary.
+
+======================
+Deployment First Steps
+======================
+
+Deploying Mattermost Enterprise is not a small project. Depending on your use-case multiple physical machines have to be set up with Mattermost server, a proxy and a database while thousands of users need to be imported through AD/FS. While we strive to make this as easy as possible it will take time and iteration.
```

Comment:
```suggestion
Deploying Mattermost Enterprise is not a small project. Depending on your use case multiple physical machines have to be set up with Mattermost server, a proxy, and a database while thousands of users need to be imported through AD/LDAP. While we strive to make this as easy as possible it will take time and iteration.
```
Confirm: Whether it's meant to be AD/FS or AD/LDAP?